test/test/test_hash_multiwriter.c

```diff
@@ -112,17 +112,20 @@ test_hash_multiwriter(void)
 
 	const void *next_key;
 	void *next_data;
-	uint32_t iter = 0;
 
 	uint32_t duplicated_keys = 0;
 	uint32_t lost_keys = 0;
 
+	struct rte_hash_iterator_state state;
+
 	snprintf(name, 32, "test%u", calledCount++);
 	hash_params.name = name;
 
 	handle = rte_hash_create(&hash_params);
 	RETURN_IF_ERROR(handle == NULL, "hash creation failed");
 
+	rte_hash_iterator_init(handle, &state);
```

Comment:
Replace the line above with:
`RETURN_IF_ERROR(rte_hash_iterator_init(handle, &state) != 0, "initialization of the hash iterator failed");`